Sources/FoundationEssentials/ErrorCodes+POSIX.swift

```diff
@@ -394,469 +62,603 @@ extension POSIXErrorCode : _ErrorCodeProtocol {
 
 @available(macOS 10.10, iOS 8.0, watchOS 2.0, tvOS 9.0, *)
 extension POSIXError {
-    public static var EPERM: POSIXErrorCode {
-        return .EPERM
+    /// Operation not permitted.
+    public static var EPERM: POSIXErrorCode { 
+        return .EPERM 
     }
 
     /// No such file or directory.
-    public static var ENOENT: POSIXErrorCode {
-        return .ENOENT
+    public static var ENOENT: POSIXErrorCode { 
```

Comment:
Looks like this might have added some trailing spaces at the end of these lines, was that intentional?

Reply:
Nope, will fix.